src/rdkafka.h

```diff
@@ -4678,6 +4678,23 @@ int rd_kafka_event_log (rd_kafka_event_t *rkev,
 			const char **fac, const char **str, int *level);
 
 
+/**
+ * @brief Extract log debug context from event.
+ *
+ * Event types:
+ *  - RD_KAFKA_EVENT_LOG
+ *
+ *  @param dst destination string for comma separated list, or NULL if using ctx_names.
```

Comment:
I think we should only provide one of these output methods, it is simple enough for the application to convert from one or the other.
Since most debugs are with a single context I think the CSV string will be the most useful.